### ISSUE TITLE
imagenet issue

### ISSUE THREAD

**Issue by Yxt1212** [CHENGY12/PLOT#3]
Thanks for your great work!
When I run your code in the imagenet dataset,  i  meet a problem,

exception: no description
  File "/Dassl.pytorch-master/dassl/data/data_manager.py", line 46, in build_data_loader
    assert len(data_loader) > 0
  File "/Dassl.pytorch-master/dassl/data/data_manager.py", line 129, in __init__
    test_loader = build_data_loader(

It can not creat the test set.I have follow the CoOp to creat the imagenet dataset,which have  "train" folder and "val" folder.

**Comment by CHENGY12** [comment]
Hi @Yxt1212 , thanks for your attention to our work. We just download the current released version and successfully run it on the imagenet dataset. Could you please check the followings:
1) the dataset path in the bash file:   for example /home/xxx/data/
2) the physical dataset path: for example /home/xxx/data/imagenet/images/(train/val/test)
3) the split file:   /home/xxx/data/imagenet/preprocessed.pkl and /home/xxx/data/imagenet/classname.txt
4) If it works, you will get the split files like /home/xxx/data/imagenet/split_fewshot
5) If all of them are correct, please check the imagenet is downloaded correctly: do you have 1000 folders in the val folder such as /home/xxx/data/imagenet/images/val/n02488291

Please don't hesitate to contact me if you have any questions.

**Comment by Yxt1212** [comment]
Thank you very much for your reply.
Problem solved！
I deleted this file“preprocessed.pkl”, and after running the code to regenerate a new file“preprocessed.pkl”, the code can run normally. But I don't understand why.
